prow/jobs/kyma/kyma-integration.yaml

```diff
@@ -19,19 +19,24 @@ presubmits: # runs on PRs
       rerun_command: "/test kyma-gke-integration"
       context: prow/kyma/kyma-gke-integration
       optional: true
-      skip_report: true # from documentation: SkipReport skips commenting and setting status on GitHub.
+      skip_report: true
       max_concurrency: 10
       labels:
-        preset-sa-vm-kyma-integration: "true"
+        preset-sa-gke-kyma-integration: "true"
       decorate: true
       path_alias: github.com/kyma-project/kyma
+      extra_refs:
+        - org: kyma-project
```

Comment:
Note to reviewer: There's "the chicken or the egg" problem here since this job gets sources from test-infra/master branch but refers to the script `pr-gke-integration.sh` that is added in **this** PR (obviously it's not on master)
To test it, just replace the `org` and `base_ref` with values valid for this PR (owner, branch)